src/others/observer/lv_observer.c

```diff
@@ -21,10 +21,17 @@
 /**********************
  *      TYPEDEFS
  **********************/
+typedef enum {
+    FLAG_COND_EQ = 0,
+    FLAG_COND_GT = 1,
+    FLAG_COND_GE = 2
+} flag_cond_t;
+
 typedef struct {
     uint32_t flag;
     lv_subject_value_t value;
-    uint32_t inv    : 1;
+    uint32_t inv     : 1;
+    flag_cond_t cond : 2;
```

Comment:
When using an enum as a bitfield on MSVC, it's a signed integer, so one extra bit is needed to represent positive 2 (`FLAG_COND_GE`). See 7d020e672fe0417049775a56b228a71c4f1bac98 from #7171

```suggestion
    flag_cond_t cond : 3;
```

Reply:
God, it must have been a nightmare to catch that error